modules/lotamePanoramaIdSystem.js

```diff
@@ -183,11 +190,50 @@ function clearLotameCache(key) {
         undefined
       );
     }
-    if (storage.hasLocalStorage()) {
+    if (localStorageIsEnabled(false)) {
       storage.removeDataFromLocalStorage(key, undefined);
     }
   }
 }
+/**
+ * @param {boolean} `honorConfig` - `false`` to override for reading or deleting old cookies
+ * @returns {boolean} for whether we can write the cookie
+ */
+function cookiesAreEnabled(honorConfig=true) {
+  if (honorConfig) {
+    return storage.cookiesAreEnabled() && appliedConfig.storage.type.includes('cookie');
+  }
+  return storage.cookiesAreEnabled();
+}
+/**
+ * @param {boolean} `honorConfig` - `false`` to override for reading or deleting old cookies
+ * @returns {boolean} for whether we can write the cookie
+ */
+function localStorageIsEnabled(honorConfig=true) {
+  if (honorConfig) {
+    return storage.hasLocalStorage() && appliedConfig.storage.type.includes('html5');
+  }
+  return storage.hasLocalStorage(); 
+}
+/**
+ * @param {SubmoduleConfig} submoduleConfig
+ * @returns {boolean} `true` - when there are errors, `false` - otherwise.
+ */
+function checkConfigHasErrorsAndReport(config) {
+  let error = false;
```

Comment:
suggest replace variable name error with hasError

Reply:
OK - that was cribbed from the Yandex solution.

Reply:
Done.